### ISSUE TITLE
conda build

### ISSUE THREAD

**Issue by lituan** [google/deepvariant#9]
Building deepvariant seems very hard, if you can put deepvariant onto anaconda cloud, then it'll be very easy for us to install.

**Comment by chapmanb** [comment]
Thanks much for suggesting this. I'm actively working on preparing a DeepVariant conda recipe for bioconda (https://github.com/bioconda/bioconda-recipes) along with the Google team. This might take a few days to get sorted out as there are a few pieces like bazel that appear to need updating in conda but I can report back here when it's available.

**Comment by MediciPrime** [comment]
@chapmanb Awesome!!  
Also, will we be forced to use Google Cloud Platform or is there a way to utilize a local server?  For example, a local server at a university?

**Comment by depristo** [comment]
@MediciPrime There's no requirement to use Google Cloud Platform right now. In what way are you being forced to use GCP today?

**Comment by MediciPrime** [comment]
@depristo thank you for taking the time to help me out.  

When I was trying to build DeepVariant on Ubuntu 16.04 it required that I install Google Cloud SDK because it needed the 'gsutil' command.  Before installing Google Cloud SDK I started reading the 'deepvariant-quick-start.md' file and it talked a lot about using Google Cloud SDK and setting up a Google Cloud account and 'enable billing for you account'.  

Based on your comment I am assuming that Google Cloud Platform is not needed and we can simply ignore 'gsutil' once the build process is complete?

**Comment by MediciPrime** [comment]
@depristo after rereading the 'deepvariant-build-test.md' file I am starting to think that 'gsutil' is being used to download the relevant files from GCP.  Anyways, I was able to successfully build it and will start fiddling with it.  Thank you for this awesome software!

**Comment by depristo** [comment]
Ah yes, that's unfortunate. We've added an internal bug to cleanup that language, as I agree it is confusing.

**Comment by rpoplin** [comment]
@MediciPrime We've updated the [quickstart guide](https://github.com/google/deepvariant/blob/r0.4/docs/deepvariant-quick-start.md) based on your feedback to make it more clear that setting up a Cloud account and enabling billing isn't required to run DeepVariant. Take the new wording in that doc out for a spin and let us know what you think.

Cheers,

**Comment by MediciPrime** [comment]
Thank you Ryan :+1: You guys are awesome!

**Comment by chapmanb** [comment]
A conda package for DeepVariant is now available through bioconda so you should be able to install with:
```
conda install -c conda-forge -c bioconda deepvariant
```
It includes wrapper scripts for each of the 3 steps (`dv_make_examples.py`, `dv_call_variants.py`, `dv_postprocess_variants.py`) that handle wrapping the internal locations of the pre-built zip files and models, so you can call these as normal command line options. These don't yet expose all options available in DeepVariant but are hopefully sufficient to run standard germline calling projects.

I've also started a separate issue (#29) to discuss improvements we can make to improve portability, but hope the initial package helps for installing and using DeepVariant.

I'd be happy for feedback and suggestions on this package as folks have a chance to use it.

**Comment by depristo** [comment]
Thank you Brad, this is super exciting! We're delighted it worked out and we'll look into the improvements you requested separately.
